app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Good Job for setting up devise to  register a new user. But when I try to register with the **email, password and password confirmation** it throws an error related with Name can't be blanked  But for the user register there is no Name input that allow the user to insert the name So kindly adjust it.
- The signup page for register the user
![image](https://github.com/martinkarugaba/blog_app/assets/93598189/9b1621e3-84c3-41ca-aabe-9e6fa8dee6da)

- The Error message showed up after I input the email , password with the password confirmation
![image](https://github.com/martinkarugaba/blog_app/assets/93598189/c5b7e02a-82e5-430c-ad1f-6a43e2016eac)